### PR TITLE
[OTA-613] - bugfix for network info

### PR DESCRIPTION
**Pull request by JustynaKK**

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/stores/DevicesStore.js [hunk 1]
```diff
@@ -223,15 +223,16 @@ export default class DevicesStore {
 
     fetchDeviceNetworkInfo(id, isFromWs = false) {
         resetAsync(this.devicesOneNetworkInfoFetchAsync, true);
-        return axios.get(API_DEVICES_NETWORK_INFO + '/' + id + '/system_info/network')
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/stores/DevicesStore.js]: Let's not change this function. It should just fetch API.

File: ota-plus-web/app/reactapp/src/stores/DevicesStore.js [hunk 2]
```diff
@@ -223,15 +223,16 @@ export default class DevicesStore {
...
-            .then((response) => {
-                this.deviceNetworkInfo = response.data;
-                this.devicesOneNetworkInfoFetchAsync = handleAsyncSuccess(response);
-            })
-            .catch((error) => {
-                this.devicesOneNetworkInfoFetchAsync = handleAsyncError(error);
-            });
-
+        if (!isFromWs || (isFromWs && this.device.uuid === id)) {
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/stores/DevicesStore.js]: Let's do this check in WebsocketHandler. Can remove `isFromWs`.

File: ota-plus-web/app/reactapp/src/utils/WebsocketHandler.js [hunk 3]
```diff
@@ -52,7 +52,7 @@ const WebsocketHandler = (function (wsUrl, stores) {
                     }
                     break;
                 case "DeviceSystemInfoChanged":
-                    stores.devicesStore.fetchDeviceNetworkInfo(data.uuid);
+                    stores.devicesStore.fetchDeviceNetworkInfo(data.uuid, {isFromWs: true});
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/utils/WebsocketHandler.js]: Check device here.

Comment (JustynaKK) [reply ota-plus-web/app/reactapp/src/utils/WebsocketHandler.js]: this is further fix to the fetching network info, related to: https://github.com/advancedtelematic/ota-plus-server/commit/042d2b311a7d80dc31d32bf9b702d3c41d4b55b2
file: ota-plus-web/app/reactapp/src/stores/DevicesStore.js, line 223

This code should stay as it is until stores refactoring.

**Pull request merged**